layouts/html.hbs

```diff
@@ -146,9 +146,31 @@
           } else if (window.isOverlay) {
             window.Overlay.onMessage(message);
           } else if (message.eventType && message.eventType === 'click') {
+            if (ANSWERS) {
+              ANSWERS.components
```

Comment:
Since this code is wrapped in the #babel HBS helper, we can use the `ANSWERS?.` syntax since it will be transpiled in the bundle